contracts/test/unit/EigenDACertVerifierV2Unit.t.sol

```diff
@@ -130,23 +132,10 @@ contract EigenDACertVerifierV2Unit is MockEigenDADeployer {
         eigenDACertVerifier.verifyDACertV2FromSignedBatch(signedBatch, blobInclusionInfo);
     }
 
-    function test_verifyDACertV2_revert_BadSecurityParams(uint256 pseudoRandomNumber) public {
```

Comment:
This is not necessary since this check is done on CertVerifier construction now.